app/models/activity.rb

```diff
@@ -60,19 +60,25 @@ def schedule=(ice_cube_schedule)
     super(ice_cube_schedule.to_yaml)
   end
 
```

Comment:
problem with slow chart load has it's source here, there is like x*366 TimeRange objects created

Reply:
Yep. Could we prewarm the caches? Or rewrite to calculate the whole graph in a better way? Don't want to prematurely optimise before anyone is actually using the site. But if there's a better way to do it we should do it

Reply:
prewarm ?

Reply:
Like run a job to rebuild caches after change. But actually `Rails.cache` just isn't working I think, see comment